docs/content/integrations/tableau.mdx

```diff
@@ -0,0 +1,128 @@
+---
+title: "Using Tableau"
```

Comment:
```suggestion
title: "Using Dagster with Tableau"
description: Represent your Tableau assets in Dagster
```

Update title based on Sandy's feedback in https://github.com/dagster-io/dagster/pull/24269